src/webui/btjson.cpp

```diff
@@ -199,7 +199,7 @@ class QTorrentCompare
         if (type_ == QVariant::Invalid)
             type_ = torrent1.toMap().value(key_).type();
 
-        switch (type_) {
+        switch (static_cast<QMetaType::Type>(type_)) {
```

Comment:
If you use cast to QMetaType::Type here you should use QMetaType consts in switch cases.